### PR TITLE
Fix pregnancy visit ODE failure when birth place doesn't exist

### PR DESCRIPTION
**Pull request by jdzak**
See https://code.bioinformatics.northwestern.edu/issues/issues/show/3489.

### CODE REVIEW

**Comment by jdzak** [comment]
I found another issue, temporarily closing the pull request until everything is working.

File: app/models/operational_data_extractor/base.rb [hunk 1]
```diff
@@ -745,43 +745,40 @@ def find_institution_type(map, type)
...
-      unless type && type.local_code != -5
-        type = find_institution_type(map, type)
-      end
-      institution = get_institution(response_set, type)
-      map.each do |key, attribute|
-        if r = data_export_identifier_indexed_responses[key]
-          value = response_value(r)
-          unless value.blank?
-            set_value(institution, attribute, value)
+      type = find_institution_type(map, type) if type && type == other_institute_type
```
Comment (hannahwhy) [inline app/models/operational_data_extractor/base.rb]: This line is confusing.   What is `type`?   

Given a value of `type` that is equivalent to `other_institute_type`, it looks like it comes in as `other_institute_type`, and then is mutated to whatever `find_institution_type` returns.  For all other values, `type` is just going to be what was passed in.

Do you just get back something equivalent to `other_institute_type` in the default case?  If so, is this just doing `type = type`?

Comment (jdzak) [inline app/models/operational_data_extractor/base.rb]: Agreed, this is kind of weird and I don't think it's even necessary.. I'll change it.

**Comment by jdzak** [comment]
Updated per @yipdw comments. Ready for review.

**Pull request merged**